roofit/roofitcore/src/RooAbsCollection.cxx

```diff
@@ -358,16 +371,16 @@ RooAbsCollection &RooAbsCollection::assignValueOnly(const RooAbsCollection& othe
     theirs->syncCache() ;
     elem->copyCache(theirs,kTRUE) ;
   }
-  return *this;
+  return ;
 }
 
 
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Functional equivalent of operator=() but assumes this and other collection
+/// Functional equivalent of assign() but assumes this and other collection
 /// have same layout. Also no attributes are copied
 
-void RooAbsCollection::assignFast(const RooAbsCollection& other, Bool_t setValDirty)
+void RooAbsCollection::assignFast(const RooAbsCollection& other, bool setValDirty)
 {
   if (&other==this) return ;
   assert(_list.size() == other._list.size());
```

Comment:
There's the new `hasSameLayout()`. It would make a lot of sense to `assert` the same layout here, not only the size.